packages/libs/react-ui/src/components/Accordion/Accordion.css.ts

```diff
@@ -13,34 +15,50 @@ export const accordionSectionClass = style([
     },
   },
 ]);
-export const accordionTitleClass = style([
-  sprinkles({
-    display: 'flex',
+
+export const accordionSectionHeadingClass = style([
+  {
     alignItems: 'center',
-    justifyContent: 'space-between',
+    background: 'none',
+    border: 'none',
+    borderBottom: `1px solid ${vars.colors.$gray20}`,
     cursor: 'pointer',
+    display: 'flex',
     fontSize: '$base',
     fontWeight: '$medium',
+    padding: 0,
+    textAlign: 'left',
+    transition: 'color 0.2s ease',
+    width: '100%',
+    selectors: {
+      '&.isOpen': {
+        border: 'none',
+      },
+    },
+  },
+]);
+
+export const accordionTitleClass = style([
+  sprinkles({
+    color: '$neutral5',
+    display: 'block',
     paddingBottom: '$2',
+    width: '100%',
   }),
-  {
-    transition: 'color 0.2s ease 0s',
-    borderBottom: '1px solid',
-  },
 ]);
-export const accordionTitleVariants = styleVariants({
-  closed: [sprinkles({ color: '$negativeContrast' })],
-  opened: [sprinkles({ color: '$foreground' })],
-});
 
-export const toggleButtonClass = style([
+export const toggleIconWrapperClass = style([
```

Comment:
I think you can combine the accordionTitleClass and toggleIconWrapperClass. You may even be able to combine it with the headingClass. I would suggest looking at ways to simplify the html and css to reduce the amount of code